cmake/modules/AddSwift.cmake

```diff
@@ -427,7 +427,7 @@ function(_add_host_variant_link_flags target)
   #
   # TODO: Evaluate/enable -f{function,data}-sections --gc-sections for bfd,
   # gold, and lld.
-  if(NOT CMAKE_BUILD_TYPE STREQUAL Debug AND CMAKE_SYSTEM_NAME MATCHES Darwin)
+  if(NOT CMAKE_BUILD_TYPE STREQUAL "Debug" AND CMAKE_SYSTEM_NAME MATCHES Darwin)
```

Comment:
Should the RHS of the `MATCHES` be quoted as well?  While a regex, it is still stringly-typed right?

Reply:
I've not seen any breakage because of this in our Nixpkgs build. `MATCHES` RHS doesn't accept `variable`, so should be safe, I believe.

However, if you'd like me to fix this up, it's easy enough to grep. There are only 18 occurrences. (And FWIW, no cases of `LINUX` or `ANDROID`.)